com.snipaste.Snipaste.yaml

```diff
@@ -0,0 +1,72 @@
+app-id: com.snipaste.Snipaste
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: snipaste
+separate-locales: false
+
+finish-args:
+  - --device=all
+
+  - --socket=wayland
+  - --socket=fallback-x11
+
+  - --share=ipc
+  - --share=network
+
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-run/pipewire-0
+
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+
+  # D-Bus Access
+  - --talk-name=org.freedesktop.Notifications
+
+modules:
+  - name: unappimage
+    buildsystem: simple
+    build-commands:
+      - make -C squashfs-tools install INSTALL_DIR=/app/bin
+    sources:
+      - type: git
+        url: https://github.com/refi64/unappimage
+        commit: d7f86f2a0d7ec3a69211125207d5f127386b849a
+
+  - name: snipaste
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 apply_extra $FLATPAK_DEST/bin/apply_extra
+      - install -Dm755 snipaste.sh $FLATPAK_DEST/bin/snipaste
+      - install -Dm644 com.snipaste.Snipaste.png -t /app/share/icons/hicolor/512x512/apps
+      - install -Dm644 com.snipaste.Snipaste.desktop -t /app/share/applications
+      - install -Dm644 com.snipaste.Snipaste.metainfo.xml -t /app/share/metainfo/
+    sources:
+      - type: script
+        commands:
+          - chmod +x Snipaste.AppImage
+          - unappimage Snipaste.AppImage
+          - mkdir -p /app/extra/bin
+          - mv squashfs-root/usr /app/extra/bin/snipaste
+          - rm -rf Snipaste.AppImage squashfs-root
+        dest-filename: apply_extra
+
+      - type: script
+        commands:
+          - exec zypak-wrapper /app/extra/bin/snipaste/bin/Snipaste "$@"
+        dest-filename: snipaste.sh
+
+      - type: file
+        path: com.snipaste.Snipaste.png
+
+      - type: file
+        path: com.snipaste.Snipaste.desktop
+
+      - type: file
+        path: com.snipaste.Snipaste.metainfo.xml
+
+      - type: extra-data
+        only-arches: [x86_64]
+        filename: Snipaste.AppImage
+        url: https://bitbucket.org/liule/snipaste/downloads/Snipaste-2.10.5-x86_64.AppImage
```

Comment:
why are these coming from a bitbucket and not from `snipaste.com`?

Reply:
Because Snipaste distribute packages from bitbucket.

Reply:
Because links from snipaste.com always redirect to the latest version rather than to a specific version.